mlrun/feature_store/retrieval/base.py

```diff
@@ -31,6 +31,8 @@
 class BaseMerger(abc.ABC):
     """abstract feature merger class"""
 
+    support_online = False
+    support_offline = False
```

Comment:
Can you add a comment here explaining what each of those mean, and what functions need to be implemented for each?